### PR TITLE
Nightmare

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Added Nightmares. They're admin only currently, so as usual, make sure to beg admins to be one.
/:cl: 

Nightmares

-Burn in the light/heal in the dark

-Have nightvision

-Have shadow walk

-Have an armblade that can destroy objects that produce light, and has a chance to destroy light producing objects a targeted mob is holding. It will also deactivate borg headlamps for 10 seconds.

-Can't be hit by gunfire in the dark (this is my sad attempt at solving the stun issue, but they'll likely need something else)

Happy headmin day @Armhulen 

I also did a minor refactor of borgs losing their ability to use headlamps to use world.time instead of a timer

### CODE REVIEW

**Comment by Repukan** [comment]
Make their converts NPCs Ala zombie blobs so it isn't that overpowered and makes people not run to get converted?

Make them shadowdemons or something that spread shadowshrooms every x amount of steps, maybe.

**Comment by KorPhaeron** [comment]
@Repukan 

My idea for conversion was going to be like alien infection that progresses when they're in the dark, and an evil shadow thing pops out if they dont get the tumour removed fast enough (and someone else plays that shadow thing)

**Comment by Repukan** [comment]
Sounds cool but then you'd have to build the sling into something that could defend a nest (Since queens and praets are built for that)

Unless it's ridiculously quick but then the infected would be fucked nonetheless due to how slow surgery is.

**Comment by KorPhaeron** [comment]
Guess I'll have to figure that out. However it works the victim wont be the one controlling the new mob to discourage """"accidentally"""" getting caught

**Comment by pubby** [comment]
What's their objective gonna be?

**Comment by KorPhaeron** [comment]
Eat a bunch of peoples hearts or X tiles of darkness or something spooky dunno yet

**Comment by Armhulen** [comment]
WOOOOOOOOOOOOOOOOOO

**Comment by Xhuis** [comment]
Aw heck, it's really happening. If you need me for anything lemme know.

**Comment by PraiseRatvar** [comment]
Not to act like a party pooper but these sound ALOT like slings.

**Comment by vcordie** [comment]
:thinking:

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 1]
```diff
@@ -21,3 +22,60 @@
...
+/datum/species/shadow/nightmare/bullet_act(obj/item/projectile/P, mob/living/carbon/human/H)
+	var/light_amount = 0
+	if(isturf(H.loc))
+		var/turf/T = H.loc
+		light_amount = T.get_lumcount()
+		if (light_amount < 0.2)
+			H.visible_message("<span class='danger'>[H] dances in the shadows, evading [P]!</span>")
+			playsound(T, pick('sound/weapons/bulletflyby.ogg', 'sound/weapons/bulletflyby2.ogg', 'sound/weapons/bulletflyby3.ogg'), 75, 1)
+			return -1
+	return 0
```
Comment (Xhuis) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: I'd replace this and `return -1` with just `return`, unless `return -1` is explicitly checked somehow

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 2]
```diff
@@ -21,3 +22,60 @@
...
+
+/obj/item/light_eater
+	name = "light eater"
+	icon_state = "arm_blade"
+	item_state = "arm_blade"
+	force = 25
```
Comment (Xhuis) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: A permanent armblade seems a little... weird. But hey!

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 3]
```diff
@@ -21,3 +22,60 @@
...
+			if(O.light_power)
+				disintegrate(O)
+	if(isitem(AM))
+		var/obj/item/I = AM
+		if(I.light_power)
+			disintegrate(I)
+
+/obj/item/light_eater/proc/disintegrate(obj/item/O)
+	if(prob((200 / O.light_range)))
+		src.loc.visible_message("<span class='danger'>[O] is disintegrated by [src]!</span>")
```
Comment (Xhuis) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: `loc.visible_message` is probably cleaner here

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 4]
```diff
@@ -21,3 +22,60 @@
...
 			H.heal_overall_damage(1,1)
+
+
+/datum/species/shadow/nightmare
+	name = "Nightmare"
+	id = "nightmare"
+	burnmod = 1.5
+	blacklisted = TRUE
+	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOGUNS,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE,NODISMEMBER,NO_UNDERWEAR)
+	mutanteyes = /obj/item/organ/eyes/night_vision/nightmare
```
Comment (Xhuis) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: I think @FuryMcFlurry made some new shadowling race sprites, so if you'd like to use those they'd look very much better than the old.

**Review by ghost — COMMENTED** [review]
As the person who made shadowling and failed to revive it, I feel obligated to help out here.

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm (hunk 1 above)
Comment (KorPhaeron) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: It is

**Comment by KorPhaeron** [comment]
@PraiseRatvar 

Shadowlings are a conversion mode and these are going to be balanced to fit into regular play

These have also lost all their abilities with 0 counterplay (AoE death freeze, ranged parasting, unstoppable jaunt)

**Comment by JJRcop** [comment]
To keep them limited in destruction, make enthrallment temporary.

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm (hunk 1 above)
Comment (Xhuis) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: Oh, neat. Okey doke.

**Comment by MoreRobustThanYou** [comment]
I like it

**Comment by KorPhaeron** [comment]
@Xhuis 

Also regarding the sprites I'm still not sure yet, if @MoreRobustThanYou plans on porting this downstream it might make sense to use different sprites so it can coexist with regular shadowlings on their server

**Comment by Xhuis** [comment]
Alright, that works

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 5]
```diff
@@ -21,3 +22,62 @@
...
+			if(O.light_power)
+				disintegrate(O)
+	if(isitem(AM))
+		var/obj/item/I = AM
+		if(I.light_power)
+			disintegrate(I)
+
+/obj/item/light_eater/proc/disintegrate(obj/item/O)
+	src.loc.visible_message("<span class='danger'>[O] is disintegrated by [src]!</span>")
+	playsound(src.loc, 'sound/items/welder.ogg', 50, 1)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: KORRRRRRRR!!!

Comment (Cyberboss) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: O.visible_message

Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: no src.

Comment (Cyberboss) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: playsound(src

**Comment by TechnoAlchemisto** [comment]
Wow, this looks great! I remember when we were in school I always saw you in the computer lab making your games, this is awesome! Keep up the good work!

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 6]
```diff
@@ -21,3 +22,62 @@
...
+	return 0
+
+/obj/item/light_eater
+	name = "light eater"
+	icon_state = "arm_blade"
+	item_state = "arm_blade"
+	force = 25
+	lefthand_file = 'icons/mob/inhands/antag/changeling_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/antag/changeling_righthand.dmi'
+	flags_1 = ABSTRACT_1 | NODROP_1
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: Should probably have DROPDEL_1 too for sanity

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 7]
```diff
@@ -21,3 +22,62 @@
...
+	w_class = WEIGHT_CLASS_HUGE
+	sharpness = IS_SHARP
+
+/obj/item/light_eater/afterattack(atom/movable/AM, mob/user, proximity)
+	if(!proximity)
+		return
+	if(isopenturf(AM)) //So you can actually melee with it
+		return
+	if(isliving(AM))
+		for(var/obj/item/O in AM)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: I think you just want /obj

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 8]
```diff
@@ -21,3 +22,62 @@
...
+			if(O.light_power)
+				disintegrate(O)
+	if(isitem(AM))
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: again, just /obj

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 9]
```diff
@@ -21,3 +22,62 @@
...
+		var/obj/item/I = AM
+		if(I.light_power)
+			disintegrate(I)
+
+/obj/item/light_eater/proc/disintegrate(obj/item/O)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: again, just /obj

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 10]
```diff
@@ -21,3 +22,62 @@
...
+
+
+/datum/species/shadow/nightmare/bullet_act(obj/item/projectile/P, mob/living/carbon/human/H)
+	var/light_amount = 0
+	if(isturf(H.loc))
+		var/turf/T = H.loc
+		light_amount = T.get_lumcount()
+		if (light_amount < 0.2)
+			H.visible_message("<span class='danger'>[H] dances in the shadows, evading [P]!</span>")
+			playsound(T, pick('sound/weapons/bulletflyby.ogg', 'sound/weapons/bulletflyby2.ogg', 'sound/weapons/bulletflyby3.ogg'), 75, 1)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: use get_sfx

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 11]
```diff
@@ -21,3 +22,62 @@
...
+	var/obj/effect/proc_holder/spell/targeted/shadowwalk/SW = new
+	C.AddSpell(SW)
+	var/obj/item/light_eater/blade = new
+	C.put_in_hands(blade)
+
+
+/datum/species/shadow/nightmare/bullet_act(obj/item/projectile/P, mob/living/carbon/human/H)
+	var/light_amount = 0
+	if(isturf(H.loc))
+		var/turf/T = H.loc
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: var/turf/T = H.loc
if(istype(T))

**Comment by ChangelingRain** [comment]
As for the stun issue I think they should just be stun immune. Flat out 'fuck your baton' stun immune, since if you can stun it, it's dead and nonthreatening.
I'd kind of suggest making them visibly not human-shaped/human, too.

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 12]
```diff
@@ -21,3 +22,62 @@
...
+/datum/species/shadow/nightmare/on_species_gain(mob/living/carbon/C, datum/species/old_species)
+	. = ..()
+	var/obj/effect/proc_holder/spell/targeted/shadowwalk/SW = new
+	C.AddSpell(SW)
+	var/obj/item/light_eater/blade = new
+	C.put_in_hands(blade)
+
+
+/datum/species/shadow/nightmare/bullet_act(obj/item/projectile/P, mob/living/carbon/human/H)
+	var/light_amount = 0
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: The declaration isn't needed here

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 13]
```diff
@@ -21,3 +22,62 @@
...
+	if(isturf(H.loc))
+		var/turf/T = H.loc
+		light_amount = T.get_lumcount()
+		if (light_amount < 0.2)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: define

**Review by Cyberboss — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm (hunk 8 above)
Comment (KorPhaeron) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: I dont want them disintegrating machines and structures in a single hit

**Comment by KorPhaeron** [comment]
@ChangelingRain 

Yeah but then it picks up a baton and just abuses stun immunity to win the videogame and nobody is having fun anymore

**Comment by Iamgoofball** [comment]
Xenomorph reskin

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm (hunk 7 above)
Comment (KorPhaeron) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: I dont want them disintegrating machines and structures in a single hit

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm (hunk 9 above)
Comment (KorPhaeron) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: I dont want them disintegrating machines and structures in a single hit

**Comment by ChangelingRain** [comment]
then don't give it hands!

**Comment by ChangelingRain** [comment]
just turn it into a simple animal totally dodging stun issues is worth it

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 14]
```diff
@@ -42,13 +41,12 @@
...
-		var/turf/T = H.loc
-		light_amount = T.get_lumcount()
-		if (light_amount < 0.2)
+	var/turf/T = H.loc
+	if(istype(T))
+		var/light_amount = T.get_lumcount()
+		if(light_amount < SHADOW_SPECIES_LIGHT_THRESHOLD)
 			H.visible_message("<span class='danger'>[H] dances in the shadows, evading [P]!</span>")
-			playsound(T, pick('sound/weapons/bulletflyby.ogg', 'sound/weapons/bulletflyby2.ogg', 'sound/weapons/bulletflyby3.ogg'), 75, 1)
+			playsound(T, get_sfx("bullet_miss"), 75, 1)
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: you can remove the get_sfx() since playsound calls it

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 15]
```diff
@@ -70,14 +69,14 @@
 		return
 	if(isliving(AM))
 		for(var/obj/item/O in AM)
-			if(O.light_power)
+			if(O.light_range)
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: it should check both

**Comment by MoreRobustThanYou** [comment]
or just make it unable to use guns or batons

**Comment by calzilla1** [comment]
Make the converts grues

**Comment by DoctoreMad** [comment]
Do you even know how a grue looks like?

**Comment by Xhuis** [comment]
Big, angry, with giant scary teeth!

File: code/modules/surgery/organs/eyes.dm [hunk 16]
```diff
@@ -61,14 +61,17 @@
 /obj/item/organ/eyes/night_vision/alien
 	name = "alien eyes"
 	desc = "It turned out they had them after all!"
-	see_in_dark = 8
-	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
```
Comment (Cyberboss) [inline code/modules/surgery/organs/eyes.dm]: Why this change?

Comment (KorPhaeron) [reply code/modules/surgery/organs/eyes.dm]: Duplicated lines, it inherits that already

**Comment by XDTM** [comment]
I think that making convertees easy to tell apart if analyzed (like the old thralls with odd eyes, if they weren't wearing glasses) would heavily reduce sec death camps.
Also, making them very expendable and with zero free will would set them apart from other conversion modes.

**Comment by ChangelingRain** [comment]
Or make them converts in the sense that cult constructs are converts; sure, it used to be a person, but now it's trying to kill you.

**Comment by Iamgoofball** [comment]
Xenomorph cult reskin

**Comment by Ratvar** [comment]
Ability to create impassible wall in darkness that dies off if anyone uses light/welder on it instead of glare? No stunning your target, but can prevent it escaping when you kill all lights and make your own homelands. Some resprited weak resins. Medium cooldown, will help with trapping alone people but will die to anyone with PDA lights on. Will also help to trap "hatching" infecties.

**Comment by Ratvar** [comment]
How they gonna fuck glowshrooms? Ability to turn patches into shadowshroom? Cannot overcome shitton of glowies, people gonna burn it themself, and, when you find some-they are already everywhere. And it also burns you when you try to walk near. And it reveals where sling is.

**Comment by KorPhaeron** [comment]
@Ratvar

Their armblade destroys glowshrooms

**Comment by ChangelingRain** [comment]
It does, yes, but not instantly so.

**Comment by FuryMcFlurry** [comment]
These are using my sprites right?

**Comment by sawrge** [comment]
rename them demonlords and make the darkness only spread in one direction

**Comment by Armhulen** [comment]
15 magical barriers

**Comment by ChangelingRain** [comment]
if you want monster designs for this to turn people into, here's some:

- mostly-invisible monster that can attack any visible target that is in at least moderate darkness. those attacks apply an increasing level of darkness to the target. can walk freely through walls, but releases visible smoke in lit areas as it takes damage.
- strong, slow monster that has only human-level health, but emits a darkness aura and multiplies damage taken based on light level, so it's much harder to kill in low light.

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 17]
```diff
@@ -68,12 +68,18 @@
...
-				disintegrate(O)
+		if(iscyborg(AM))
+			var/mob/living/silicon/robot/borg = AM
+			borg.update_headlamp(TRUE, 100)
+			return
+		else
+			for(var/obj/item/O in AM)
+				if(O.light_range && O.light_power)
+					disintegrate(O)
+			return
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: remove these returns and make the if(isitem(AM)) into an else if

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 18]
```diff
@@ -68,12 +68,18 @@
 	if(isopenturf(AM)) //So you can actually melee with it
 		return
 	if(isliving(AM))
-		for(var/obj/item/O in AM)
-			if(O.light_range)
-				disintegrate(O)
+		if(iscyborg(AM))
+			var/mob/living/silicon/robot/borg = AM
+			borg.update_headlamp(TRUE, 100)
+			return
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: yes, both returns

**Review by ChangelingRain — DISMISSED** [review]

**Comment by DoctoreMad** [comment]
Clearly, we'll need to give a 10,000 LED water cooled flashlight for the deathsquad.

**Comment by ExcessiveUseOfCobblestone** [comment]
@sawrge moving left of a demon causes brute damage and necropolis curse

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm (hunk 4 above)
Comment (Armhulen) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: i think we've been here

**Comment by KorPhaeron** [comment]
@Cyberboss It's playable and I think fun and I think everything you wanted changed was addressed, I'll add objectives/automated spawning etc in another PR

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 19]
```diff
@@ -7,17 +7,89 @@
...
+	C.put_in_hands(blade)
+
+
+/datum/species/shadow/nightmare/bullet_act(obj/item/projectile/P, mob/living/carbon/human/H)
+	var/turf/T = H.loc
+	if(istype(T))
+		var/light_amount = T.get_lumcount()
+		if(light_amount < SHADOW_SPECIES_LIGHT_THRESHOLD)
+			H.visible_message("<span class='danger'>[H] dances in the shadows, evading [P]!</span>")
+			playsound(T, get_sfx("bullet_miss"), 75, 1)
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: since playsound calls get_sfx() you can remove the call here

**Comment by KorPhaeron** [comment]
@ChangelingRain 

Forgot about that one in my rushed attempts to fix PDA vaporization sorry

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 20]
```diff
@@ -7,17 +7,89 @@
...
+	limbs_id = "shadow"
+	burnmod = 1.5
+	blacklisted = TRUE
+	no_equip = list(slot_wear_mask, slot_wear_suit, slot_gloves, slot_shoes, slot_w_uniform, slot_s_store)
+	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOGUNS,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE,NODISMEMBER,NO_UNDERWEAR,NOHUNGER)
+	mutanteyes = /obj/item/organ/eyes/night_vision/nightmare
+
+/datum/species/shadow/nightmare/on_species_gain(mob/living/carbon/C, datum/species/old_species)
+	. = ..()
+	var/obj/effect/proc_holder/spell/targeted/shadowwalk/SW = new
```
Comment (AnturK) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: These need to be removed on species change. Also there was specific variable for species inhands. Zombies used that, you should probably use/repurpose that if possible.

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 21]
```diff
@@ -7,17 +7,89 @@
...
+	sharpness = IS_SHARP
+
+/obj/item/light_eater/afterattack(atom/movable/AM, mob/user, proximity)
+	if(!proximity)
+		return
+	if(isopenturf(AM)) //So you can actually melee with it
+		return
+	if(isliving(AM))
+		if(iscyborg(AM))
+			var/mob/living/silicon/robot/borg = AM
```
Comment (AnturK) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: Just make this generic function , extinguish_light or similar instead of all these istypes.

Comment (KorPhaeron) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: Is an atom level proc for one weapon better than a handful of istypes?

File: code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 22]
```diff
@@ -7,17 +7,98 @@
...
+
+
+/datum/species/shadow/nightmare
+	name = "Nightmare"
+	id = "nightmare"
+	limbs_id = "shadow"
+	burnmod = 1.5
+	blacklisted = TRUE
+	no_equip = list(slot_wear_mask, slot_wear_suit, slot_gloves, slot_shoes, slot_w_uniform, slot_s_store)
+	species_traits = list(NOBREATH,RESISTCOLD,RESISTPRESSURE,NOGUNS,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,PIERCEIMMUNE,NODISMEMBER,NO_UNDERWEAR,NOHUNGER)
```
Comment (XDTM) [inline code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: Can't wait for a ling to trans sting everyone with this

Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/carbon/human/species_types/shadowpeople.dm]: The Darkness will CONSUME you!

**Comment by Ratvar** [comment]
What if you make them Legion/Collector Xcuis thing lavaland final boss looking, black and purple theme of corruption, moving a bit away from shadowpeps?

**Comment by PKPenguin321** [comment]
Old shadowling was fun in spite of its flaws but sure let's just give up and make it a side antag 😟

**Comment by Repukan** [comment]
Fuck no it wasn't.

Stop seeing everything with the same rose tinted glasses bullshit that brought us the god awful gang cycle.

**Comment by KorPhaeron** [comment]
@PKPenguin321 

I don't think it's giving up when everyone who has played it so far has said they've liked it more than shadowling

**Comment by Xhuis** [comment]
It's definitely more unique than old shadowling imo and has a lot of potential even in its stripped-down state

**Comment by JJRcop** [comment]
KorPhaeron/kortg#29

**Comment by KorPhaeron** [comment]
@JJRcop 

I'm not sure what the goal there is

**Comment by JJRcop** [comment]
If the lighting is red then they have a bit more leniency when it comes to shadowwalk.

**Comment by pubby** [comment]
JJRcop is a cop :policeman: 

it breaks his immersion if antagonists cannot run red lights

**Comment by KorPhaeron** [comment]
@tgstation/commit-access 

I think this is done for now

**Comment by KorPhaeron** [comment]
@JJRcop I know what it does I'm just not sure why your'e doing it or if it will be intuitive to players

**Comment by JJRcop** [comment]
Maybe it isn't, I mostly just thought it would be interesting, because red light is scarier.

**Comment by Xhuis** [comment]
Aw heck

**Comment by optimumtact** [comment]
I was gonna close it with lol for old times sake
